04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,44 @@
 #write your code here
+
+# the array 0 index should be moved to a negative index
+# divide multiple words up into separate indices in an array, 
+# pig latinize them, then recombine into a string
+
+#if first letter is not a vowel, keep checking until it is
+# EXCEPT combine "qu"
+# everything before the vowel gets moved to the back
+# add "ay" onto the end array[i] edited + "ay"
+
+# Can I create a new string with the cut out beginning letters and push to end of array index?
+
+
+#if array 0-3 include sch or qu or ch
+
+def translate(string)
```

Comment:
Also the spacing/indentation is little off, we don't need the spacing between `if` and `else` statements because their contiguous alignment effectively communicates their logical association.